src/components/optimizationSetup/ThermalSetupWIndow.tsx

```diff
@@ -0,0 +1,108 @@
+import React from "react";
+import { useState } from "react";
+import "normalize.css";
+import "@blueprintjs/core/lib/css/blueprint.css";
+import "@blueprintjs/icons/lib/css/blueprint-icons.css";
+import "./ThermalSetupWindow.css";
+import {
+    Button,
+    FocusStyleManager,
+    NumericInput,
+    Tag,
+    Divider
+  } from "@blueprintjs/core";
+
+FocusStyleManager.onlyShowFocusOnTabs();
+
+const ThermalSetupWindow: React.FC = () => {
+    const [tableElements, setTableElements] = useState([]);
+    const [numTableElements, setNumTableElements] = useState(0); // These are currently unused
+
+    const handleAddDevice = () => {
+        
+    };
+    const handleRemoveDevice = () => {
+        
+    };
+    
+    return (
+        <div className="mainDiv">
+            <div className="thermalSetupText">Thermal Setup</div>
```

Comment:
The title here for 'Thermal Setup' isn't needed and can be removed, since the popup that this component is being called in already has the title.